packages/react-vapor/src/components/table-hoc/tests/TableHOCReducers.spec.ts

```diff
@@ -0,0 +1,37 @@
+import _ from 'underscore';
+import {IReduxAction} from '../../../utils/ReduxUtils';
+import {TableHOCActions, TableHOCSetEmptyStatePayload} from '../actions/TableHOCActions';
+import {HOCTableState, TableHOCReducers} from '../reducers/TableHOCReducers';
+
+describe('TableHOCReducers', () => {
+    const fakeAction: IReduxAction<TableHOCSetEmptyStatePayload> = {
+        type: 'FAKE',
+        payload: {
+            id: 'fake-table',
+            emptyStateSet: false,
+        },
+    };
+
+    it('should return the default state if the action is not defined', () => {
+        const oldState: HOCTableState[] = undefined;
+        const tablesHOCState: HOCTableState[] = TableHOCReducers(oldState, fakeAction);
+
+        expect(tablesHOCState).toEqual([]);
+    });
+
+    it('should return the old state when the action is not defined', () => {
+        const oldState: HOCTableState[] = [{id: 'a-table', emptyStateSet: true}];
+        const tablesHOCState: HOCTableState = TableHOCReducers(oldState, fakeAction);
+
+        expect(tablesHOCState).toBe(oldState);
+    });
```

Comment:
Your two test titles here basically have the same condition `if the action is not defined` `when the action is not defined`.

So you are basically saying that two different things can happen with the same condition, which is confusing.  I guess this has something to do with the oldState being defined or not. IMO you should specify that.